templates/server-statefulset.yaml

```diff
@@ -64,6 +72,8 @@ spec:
           command: {{ template "vault.command" . }}
           args: {{ template "vault.args" . }}
           env:
+            - name: VAULT_LOG_LEVEL
+              value: debug
```

Comment:
This needs to be removed.

Reply:
Thanks, fixed with latest commit. This was actually the reason why the previous unit tests were failing(the ones that were checking environment variables based on indexes)